students/seungmoo-lee/account/views.py

```diff
@@ -0,0 +1,48 @@
+import json
+from json.decoder import JSONDecodeError
+
+from django.http.response import JsonResponse
+from django.views         import View
+from django.db.models     import Q
+
+from .models              import User
+from .validations         import UserValidation
+
+class SignUpView(View):
+    def post(self, request):
+        user_validation = UserValidation()
+
+        try:
+            data         = json.loads(request.body)
+            email        = data.get('email')
+            password     = data.get('password')
+            phone_number = data.get('phone_number')
+            nickname     = data.get('nickname')
+
+            if user_validation.check_required_fields(email, password):
+                return JsonResponse({'message': 'KEY_ERROR'}, status=400)
+            if user_validation.check_email(email):
+                return JsonResponse({'message': 'EMAIL_ERROR'}, status=400)
+            if user_validation.check_password(password):
+                return JsonResponse({'message': 'PASSWORD_ERROR'}, status=400)
```

Comment:
```suggestion
            if user_validation.check_required_fields(email, password):
                return JsonResponse({'message': 'KEY_ERROR'}, status=400)
                
            if user_validation.check_email(email):
                return JsonResponse({'message': 'EMAIL_ERROR'}, status=400)
                
            if user_validation.check_password(password):
                return JsonResponse({'message': 'PASSWORD_ERROR'}, status=400)
```
가독성 측면에서 위와 같이 하는 것이 더 나을 것 같습니다.